drivers/i2s/i2s_cavs.c

```diff
@@ -766,6 +771,7 @@ static void i2s1_irq_config(void)
 
 static const struct i2s_cavs_config i2s1_cavs_config = {
 	.regs = (struct i2s_cavs_ssp *)SSP_BASE(1),
+	.mn_regs = &((struct i2s_cavs_mn_div *)SSP_MN_DIV_BASE)[1],
```

Comment:
It will translate into the address that is 8 bytes higher than `SSP_MN_DIV_BASE` because `sizeof(struct i2s_cavs_mn_div)` is 8 bytes.

Let me know if you want me to change this to be consistent with the `SSP_BASE(1)` statement one line above

Reply:
Yes. Let's try to keep this uniform. Please change it to be like SSP_BASE(1).

Reply:
Changed.